app/controllers/collections_controller.rb

```diff
@@ -45,6 +45,8 @@ def create
                                       release_duration: @collection_form.release_duration,
                                       access: @collection_form.access,
                                       doi_option: @collection_form.doi_option,
+                                      license_option: @collection_form.license_option,
```

Comment:
If I understand correctly, many of the values being set here are updated in the DepositCollectionJob. If that is the case, then only need to create the minimal Collection here.